<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>partSettingsSender</class>
 <widget class="QWidget" name="partSettingsSender">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QCheckBox" name="chkUseEndCommands">
       <property name="text">
        <string>Use program end commands:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkUseStartCommands">
       <property name="text">
        <string>Use program start commands:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="txtAfterPauseCommands">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="chkUseToolChangeCommands">
       <property name="text">
        <string>Use tool change commands:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QPlainTextEdit" name="txtToolChangeCommands">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPlainTextEdit" name="txtBeforePauseCommands">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chkUsePauseCommands">
       <property name="text">
        <string>Use pause commands</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPlainTextEdit" name="txtStartCommands">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPlainTextEdit" name="txtEndCommands">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkConfirmToolChangeCommandsExecution">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Confirm tool change commands execution</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkPauseOnToolChange">
     <property name="text">
      <string>Pause sender on tool change (M6) command</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkIgnoreResponseErrors">
     <property name="text">
      <string>Ignore error responses</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSetParseStateBeforeSendFromLine">
     <property name="text">
      <string>Automatically set parser state before sending from selected line</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
